src/Api.elm

```diff
@@ -0,0 +1,96 @@
+module Api exposing (..)
```

Comment:
elm-review provides an easy way for automated code cleanness checking, which is beneficial in general. Though, I've seen instances - and guilty of - engaging in dragged out arguments about which rules to enable with similar code cops.

Reply:
Explicit imports - depends on the territory. I usually unqualified import everything from core dependencies of my modules. For example, in `Decoders.elm`, I `import Json.Decode exposing (..)` because it's a core concept of the file. But in other instances, such as the case you mentioned about `Response.fromResult` I absolutely agree - there it's better to use a qualified import because `Response`s are not the core concept of the Main module.
Explicit exports - I've considered it, but because of time constraints I wanted to be able to move swiftly while coding so I just started every module with `exposing (..)` and kinda left it there. Of course with packages it becomes important to only expose what you intend to expose, but I find it less important with applications.